GitTfs/Core/IGitTfsRemote.cs

```diff
@@ -48,6 +48,7 @@ public interface IGitTfsRemote
         string Prefix { get; }
         bool ExportMetadatas { get; set; }
         Dictionary<string, string> ExportWorkitemsMapping { get; set; }
+        int? InitialChangeset { get; set; }
```

Comment:
> It should have been a method pair

Minor correction: It should _be_ a method pair

Please change this and update the pull request.